romancal/datamodels/schemas/core.schema.yaml

```diff
@@ -38,27 +45,114 @@ properties:
           start_time:
             title: UTC exposure start time
             type: number
+          mid_time:
+            title: UTC exposure mid time
+            type: number
           end_time:
             title: UTC exposure end time
             type: number
+          start_time_mjd:
+            title: "[d] exposure start time in MJD"
+            type: number
+          mid_time_mjd:
+            title: "[d] exposure mid time in MJD"
+            type: number
+          end_time_mjd:
+            title: "[d] exposure end time in MJD"
+            type: number
+          start_time_tdb:
+            title: "[d] TDB time of exposure start in MJD"
+            type: number
+          mid_time_tdb:
+            title: "[d] TDB time of exposure mid in MJD"
+            type: number
+          end_time_tdb:
+            title: "[d] TDB time of exposure end in MJD"
+            type: number
+          start_time_eng:
+            title: UTC exposure start time from engineering data
+            type: string
+          osf_file:
+            title: Observatory Status File name covering this exposure
+            type: string
+          readpatt:
+            title: Readout pattern
+            type: string
```

Comment:
Some of this will need to be derived from the MultiAccum (MA) tables, see
https://innerspace.stsci.edu/display/ROMAN/WFI+Science+Telemetry#WFIScienceTelemetry-MultiAccumTables
https://innerspace.stsci.edu/pages/viewpage.action?spaceKey=ROMAN&title=Science+Data+Formatting+Discovery
and 
https://innerspace.stsci.edu/display/ROMAN/WFI+Guide+Windows#WFIGuideWindows-GuideWindowMultiAccumTable

We will also need to store the MA table used, MA table id?

Reply:
Implemented as ma_table_name.